app/scripts/migrations/099.test.ts

```diff
@@ -0,0 +1,243 @@
+import { v4 as uuid } from 'uuid';
+import { sha256FromString } from 'ethereumjs-util';
+import { EthMethod, InternalAccount } from '@metamask/keyring-api';
+import { migrate } from './099';
+
+const MOCK_ADDRESS = '0x0';
+
+function addressToUUID(address: string): string {
+  return uuid({
+    random: sha256FromString(address).slice(0, 16),
+  });
+}
+
+interface Identity {
+  name: string;
+  address: string;
+  lastSelected?: number;
+}
+
+function createMockPreferenceControllerState(
+  identities: Identity[] = [{ name: 'Account 1', address: MOCK_ADDRESS }],
+  selectedAddress: string = MOCK_ADDRESS,
+): {
+  identities: {
+    [key: string]: { name: string; lastSelected?: number; address: string };
+  };
```

Comment:
Make a type out of this

Reply:
Changed in e71fe10d662b81a67282666672cc4402c4b9f363